core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -20,6 +22,8 @@
  */
 class ModuleSearch extends Module
 {
+	use LegacyFigureBuilderTrait;
+
```

Comment:
We should IMHO not add new code for 4.12 using this trait but rather use [FigureBuilder#buildIfResourceExists()](https://github.com/contao/contao/pull/2733).

Reply:
It's not merged yet so I couldn't use it :) We can remove that trait in all of the core once the other PR is merged.

Reply:
Done